config/locales/en.yml

```diff
@@ -25,3 +25,178 @@ en:
       %{logo_img_tag}
       <p class="lead">OnDemand provides an integrated, single access point for all of your HPC resources.</p>
     motd_title: "Message of the Day"
+    breadcrumbs:
+      home: "Home"
+      my_sessions: "My Interactive Sessions"
+      all_apps: "All Apps"
+    restart_msg_tag: |
+      Your group membership has changed, which affects your access to apps. Your dashboard should restart automatically. 
+      If it doesn't please, click: <a href="%{restart_url}">restart web server</a>.
+    nav:
+      sessions: "My Interactive Sessions"
+      all_apps: "All Apps"
+      develop:
+        title: "Develop"
+        docs: "Developer Documentation"
```

Comment:
Lets drop localizing the develop dropdown for now till we really do all of the develop dropdown (Develop, Developer Documentation, Sandbox Apps, Shared Apps, other?)

Reply:
I think that too we should hold off on work on localizing the whole Developer interface right now.

Reply:
Well... this work is actually done. Keep it. Deleting these comments.